client/src/components/AddNewProfile.js

```diff
@@ -127,10 +157,35 @@ const AddNewProfile = () => {
           onChange={handleChange}
         />
       </Form.Field>
+      <Form.Field>
+        <Dropdown
+          onChange={profileTypeChange}
+          placeholder="Select profile type"
+          fluid
+          selection
+          options={profileTypeOptions}
+        />
+      </Form.Field>
+      <Form.Field>
+        <Dropdown
+          onChange={statusChange}
+          placeholder="Select status"
+          defaultValue="new"
```

Comment:
This defaults the dropdown to `new` but doesn't default `profileData.status` to `new` which means if we don't change this dropdown and click on `Add new profile` then `profileData.status` is sent as `''` and not `new`. Let's remove this.